package.json

```diff
@@ -17,6 +17,7 @@
     "modern-normalize": "^1.1.0",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-loader-spinner": "^5.1.3",
```

Comment:
It's better not to use '^' in package.json. Strict defined versions are preffered.